modules/ROOT/pages/_partials/data-sync/server-offline-and-conflict.adoc

```diff
@@ -108,164 +64,104 @@ type Mutation {
 }
 ----
 
-. Implement the resolver.
-
-Conflicts can be resolved either on the client or the server. Depending on the strategy used, the resolver implementation will differ.
-See the sections below for the individual implementations.
-
-== Resolving Conflicts on the Client
+. Implement the resolver. Every conflict can be handled using a set of predefined steps, please see below for an example.
```

Comment:
. Implement the resolver. Every conflict can be handled using a set of predefined steps.  Please see below for an example.